app/components/Modal/Modal.css

```diff
@@ -22,28 +13,37 @@ html[data-theme='dark'] .backdrop {
 }
 
 .content {
+  line-height: 1.3;
   position: fixed;
-  max-width: 680px;
-  top: 20px;
-  left: 0;
-  right: 0;
-  margin: 0 auto;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-width: 425px;
+  width: 100%;
+  max-height: calc(100vh - 300px);
+  padding: 35px;
   background: var(--color-white);
-  border-radius: 5px;
-  padding: 40px;
   outline: none;
-  max-height: calc(100vh - 40px);
   overflow-y: auto;
   z-index: 4;
+  /* stylelint-disable indentation */
+  box-shadow: 0 5px 10px
+      rgba(var(--rgb-min), var(--rgb-min), var(--rgb-min), 12%),
+    0 6px 6px rgba(var(--rgb-min), var(--rgb-min), var(--rgb-min), 15%);
+  border: 1px solid rgba(var(--rgb-min), var(--rgb-min), var(--rgb-min), 9%);
+  border-radius: 1rem;
```

Comment:
I guess you could put the `var(--rgb-min), var(...), var(...)` in a var to make this a bit more compact and readable (sacrificing verbosity) but not that big of a deal.

Reply:
hmm I see your point, but then I wouldn't get to specify the alpha value. I don't think I can do `rgba(var(--x), y%)`. At least I believe I've tried before, but I can try again tomorrow.

Reply:
You can do
```css
--rgb-mins: var(--rgb-min), var(--rgb-min), var(--rgb-min);

border: 1px solid rgba(var(--rbg-mins), 9%);
```
I think.
but yeah it might be worse :shrug:

Reply:
ah yes, that'll work. I'm sort of  on the fence on this one. Both aren't really ideal, but I guess your option is more compact, so I think I'll change it to that.